met-api/tests/unit/services/test_submission.py

```diff
@@ -116,3 +118,25 @@ def test_submissions_with_comment_are_not_auto_approved(session):  # pylint:disa
 
     assert submission is not None
     assert submission.comment_status_id == Status.Pending.value
+
+
+def test_check_if_submission_has_comments(session):
+    """Make sure comments in submission are approved."""
+    survey, eng = factory_survey_and_eng_model()
```

Comment:
I think this test case is for a particular scenarion rite?
In that case , does the method name and the docstring reflect it?
test_check_if_submission_has_comments is very generic
Make sure comments in submission are approved. - again vague.
It will be great if either the name or the comment imply the meaning of the test.